src/components/Search.tsx

```diff
@@ -10,7 +11,7 @@ export default function Search(): JSX.Element {
   const [state, setUrl] = useHttpGet<Data>(`https://hn.algolia.com/api/v1/search?query=${query}`);
 
   return (
-    <div>
+    <Box mt={3}>
```

Comment:
Nice. I haven't used `Box` yet. I assume it just draws a box around the objects. The `mt` is mysterious, though. I'll have to go look it up. :-)

Reply:
https://material-ui.com/system/spacing/
mt is shorthand for margin-top, I love that they have this inline shorthand option too in material-ui, this is what I most missed from using bootstrap!